### DISCUSSION TITLE
3-spaces-tabs support (Support for user defined Tab Size)

### DISCUSSION THREAD

**Discussion by I-Pch** [vslinko/obsidian-outliner#60] [Archive]
As suggested by @vslinko in this comment https://github.com/vslinko/obsidian-outliner/issues/41#issuecomment-812934654 here is an idea for a future enhancement of the plugin : supporting user defined `Tab size` to indent a list 😊 (either ordered or unordered).

As of now, the plugin only support Obsidian default settings to indent a list (using `Tab`)  : `Tab size` = `4`.
(See here for more details https://github.com/vslinko/obsidian-outliner/issues/41#issuecomment-812862979)

Using a different `Tab size` (in my case it was `3`) the bullets are not recognize as fully indent and therefore not fully rendered 👇  . 

<img width="182" alt="Obs-Outliner 3" src="https://user-images.githubusercontent.com/77828085/113504711-1f421b80-953a-11eb-8652-593852c5bb9f.png">

This creates some unwanted vertical space in between the last item on the list and the previous one.

Here is what it looks like when using the Obsidian default settings `Tab size` = `4` 👇 .
(And also what I would like to get when using a different `Tab size` 😇 ). 
<img width="195" alt="Tab-size-4" src="https://user-images.githubusercontent.com/77828085/113504791-b3ac7e00-953a-11eb-8b15-1112719ee7f5.png">

I know there's no easy fix, as the problem comes from `CodeMirror` and it's kind of trivial (as it's more a visual bug than a functional one) but I personally chose a `Tab size 3` because I don't have such a big screen, so I was gaining some space (not that much, but a little bit nonetheless 😊 ) while still making my lists easily readable 😇 .  
 
That's it 😊 , that's the idea 💡 !

**Comment by vslinko** [comment]
Should be fixed in https://github.com/vslinko/obsidian-outliner/releases/tag/2.1.0
